dv-marketplace/src/components/forms/AddExternalToolForm/AddExtToolForm.tsx

```diff
@@ -0,0 +1,62 @@
+import Modal from 'react-bootstrap/Modal';
+import useAddExternalTool from './useAddExternalTool';
+
+const AddExtToolForm = () => {
+    
+
+    const {
+        handleSubmit,
+        modalIsOpen,
+        modalMessage,
+        closeModal
+    } = useAddExternalTool();
+
+    return (
+        <div className="container">
+            <h1>Add External Tool</h1>
```

Comment:
It would be nice to see some more details of why adding the tool failed:

![Screenshot 2025-03-05 at 1 49 55 PM](https://github.com/user-attachments/assets/b7a86e20-140f-4ef4-ad95-d5f6df8add34)